.github/workflows/copilot_deploy.yml

```diff
@@ -341,17 +341,17 @@ jobs:
       run_e2e_tests: ${{ inputs.run_e2e_tests || true }}
       environment: uat
 
-  production_deploy:
+  prod_deploy:
```

Comment:
Same as below - this one probably _should_ be production deploy (as it is deploying production) BUT - this is doing the copilot prod environment, so I'll just note this here for posterity